package.json

```diff
@@ -1,23 +1,44 @@
 {
-	"name": "foo-bar",
-	"title": "Foo Bar",
-	"homepage": "https://github.com/xwp/wp-foo-bar",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/xwp/wp-foo-bar.git"
-	},
-	"author": "XWP",
-	"license": "GPL-2.0+",
-	"devDependencies": {
-		"grunt": "~1.0.1",
-		"grunt-checktextdomain": "~1.0.1",
-		"grunt-contrib-clean": "~1.0.0",
-		"grunt-contrib-copy": "~1.0.0",
-		"grunt-contrib-cssmin": "~1.0.1",
-		"grunt-contrib-jshint": "~1.0.0",
-		"grunt-contrib-uglify": "~2.0.0",
-		"grunt-shell": "~1.3.1",
-		"grunt-wp-deploy": "~1.2.0",
-		"grunt-wp-i18n": "~0.5.4"
-	}
+  "name": "@xwp/wp-foo-bar",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Plugin Scaffolding for WordPress.",
+  "author": "XWP",
+  "license": "GPL-2.0-or-later",
+  "bugs": {
+    "url": "https://github.com/xwp/wp-foo-bar/issues"
+  },
+  "homepage": "https://github.com/xwp/wp-foo-bar#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xwp/wp-foo-bar.git"
+  },
+  "engines": {
+    "node": ">=10",
+    "npm": ">=6.9"
+  },
+  "scripts": {
+    "postinstall": "composer install",
+    "dev": "wp-scripts start",
+    "build": "wp-scripts build",
+    "env:start": "docker-compose up",
```

Comment:
@derekherman . Maybe `docker-compose up -d` makes more sense, otherwise `env:stop` is not too useful as the user will just `crtl+c` to exit the `docker-compose` command not running in daemon mode

Reply:
In my experience using daemon mode will sometimes create issues for me when first initiating the database, errors I don't see because all console output is hidden. I always have two shells open and manually shut down Docker with `docker-compose down` because I like to see if there are errors while building the image and running the containers. And I like to know I properly shut down the system so the port is free.

Reply:
I see. Alternatively we could also have:

```
"env:start": "docker-compose up -d",
"env:logs": "docker-compose logs -f",
"env:stop": "docker-compose stop",
```

but really i don't have a preference on that

Reply:
If we change it then there should be a callout to manually run `docker-compose up` to fully initialize. Thoughts?